concepts/05 UI Components/Gantt/05 Overview/05 Elements.md

```diff
@@ -0,0 +1,17 @@
+<img src="/images/Gantt/gantt-overview-elements.png" alt="Gantt elements" />
+
+1. [Toolbar](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/toolbar/)
+2. [Column Sort Button](/)
+3. [Header Filter](/)
+4. [Filter Row](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/filterRow/)
+5. [Selected Row](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/#allowSelection)
+6. [Task Title](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/tasks/#titleExpr)
+7. [Parent Task](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/tasks/#parentIdExpr)
+8. [Task](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/tasks/)
+9. [Task Dependency](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/dependencies/)
+10. [Resource](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/resources/)
+11. **Milestone Task**
```

Comment:
Let's use Documentation/Guide/UI_Components/Gantt/Gantt_Elements/#Task
Because only here I managed to find the description of the Milestone task